src/main/java/racingcar/domain/Race.java

```diff
@@ -1,31 +1,20 @@
 package racingcar.domain;
 
-import racingcar.domain.dto.CarRaceDto;
-import racingcar.domain.dto.RaceResultDto;
 import racingcar.domain.wrapper.RaceCount;
 
-import java.util.ArrayList;
-import java.util.List;
-
 public class Race {
 
     private final RaceCount raceCount;
 
     private Race(final String raceCount) {
-        this.raceCount = RaceCount.of(raceCount);
+        this.raceCount = RaceCount.createRaceCount(raceCount);
```

Comment:
```suggestion
        this.raceCount = RaceCount.create(raceCount);
```
정적 팩터리 메서드이니 어떤 객체를 생성하는지는 구체적으로 명시하지 않아도 괜찮아보여요.